include/z64.h

```diff
@@ -1643,6 +1643,17 @@ typedef struct ArenaNode {
     /* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
 } ArenaNode; // size = 0x30
 
+#define RELOC_SECTION(reloc) ((reloc) >> 0x1E)
```

Comment:
I've always thought it was clearer to use decimal for shifting, as the binary arrangement of bits doesn't matter here
Unless this define is standard or I'm alone on this, in which case then *sigh* so be it

```suggestion
#define RELOC_SECTION(reloc) ((reloc) >> 30)
```
reads easier to me as "get the 32-30=2 higher bits" (instead of 32-0x1E)